source/content/email.md

```diff
@@ -122,4 +122,12 @@ Please see Google's help article: [My client isn't accepting my username and pas
 
 Office 365 uses ports `25` and `587` by default, and different options for sending via SMTP client, Direct send or SMTP relay. [This document](https://docs.microsoft.com/en-us/Exchange/mail-flow-best-practices/how-to-set-up-a-multifunction-device-or-application-to-send-email-using-office-3) outlines the limitations and configurations of each, to know what will work for your application.
 
-Because we don't support SPF, it is likely that most Exchange or Office 365 servers won't work if its [configured at your email server](https://docs.microsoft.com/en-us/office365/SecurityCompliance/set-up-spf-in-office-365-to-help-prevent-spoofing). Check your organization's Microsoft Exchange settings on what's allowed by your system.
+Because we don't support SPF, it is likely that most Exchange or Office 365 servers won't work if [configured at your email server](https://docs.microsoft.com/en-us/office365/SecurityCompliance/set-up-spf-in-office-365-to-help-prevent-spoofing). Check your organization's Microsoft Exchange settings to see what's allowed by your system.
+
+### Why does my site receive numerous requests to autodiscover.xml?
```

Comment:
I suggest moving all of this content over to the "[Investigate and Remedy Traffic Events](https://pantheon.io/docs/optimize-site-traffic)" doc instead of the Email page. While autodiscover.xml is related to email, for a Pantheon customer, it's a traffic problem, not an email problem.

Reply:
I included it on both pages, email and the traffic page. Do you think that is ok?